tests/unit/test_routes.py

```diff
@@ -83,11 +83,11 @@ def test_single_resource_out_of_bounds(module_client, module_db):
 
     too_low = 0
     too_high = 9999
-    response = client.get(f"api/v1/resources{too_low}")
+    response = client.get(f"api/v1/resources/{too_low}", follow_redirects=True)
```

Comment:
The PR doesn't mention this, can you put something about why you felt this was needed?

Reply:
My second commit covers it. Basically these two tests were false positives because the url in client.get was wrong (missing a /) and returning None due to a bad url, not an empty query.

We have to use the follow_redirect because the code redirects to the 404 page.

Reply:
Oh interesting. Yes, you'll need to follow redirects to get the 404 because if you don't you get a 302. I've encountered that before elsewhere in these tests. Good catch!